ROSNET/DataModel/ROSbag.cs

```diff
@@ -1,22 +1,20 @@
-﻿using System;
-using System.IO;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 
 namespace ROSNET.DataModel
 {
+    /// <summary>
+    /// Represents a ROSbag
+    /// </summary>
     public class ROSbag
```

Comment:
Som @henrikhorluck også har nevnt, så er det lurt å lese seg litt opp på C# navnekonvensjoner. Dette ville for eksempel vært en klasse kalt RosBag